packages/dialtone-vue2/recipes/comboboxes/combobox_multi_select/combobox_multi_select.vue

```diff
@@ -613,6 +625,39 @@ export default {
         this.showValidationMessages = false;
       }
     },
+
+    setInitialInputSize () {
+      const input = this.getInput();
+      if (!input) return;
+      this.initialInputSize = input.getBoundingClientRect().height;
+      if (this.collapseOnFocusOut) {
+        this.collapseChipsWrapper();
+      }
+    },
+
+    handleInputFocusIn () {
+      if (this.collapseOnFocusOut) {
+        this.expandChipsWrapper();
+        this.setInputPadding();
+      }
+    },
+
+    handleInputFocusOut () {
+      if (this.collapseOnFocusOut) {
+        this.collapseChipsWrapper();
+        this.setInputPadding();
+      }
+    },
+
+    collapseChipsWrapper () {
+      this.$refs.chipsWrapper.style.maxHeight = `${this.initialInputSize}px`;
+      this.$refs.chipsWrapper.style.overflowY = 'hidden';
```

Comment:
I realize you were trying to make this dynamic for the different possible input sizes, but due to the issue mentioned above we are probably better off creating some new css classes that are more "fixed" and then applying those classes like so:

```
.combobox__chip-wrapper {
  max-height: initial;
  overflow-y: visible;
}
.combobox__chip-wrapper-xs--collapsed {
  max-height: 2.5rem;
  overflow-y: hidden;
}
.combobox__chip-wrapper-sm--collapsed {
  max-height: 2.7rem;
  overflow-y: hidden;
}
and so on......
```

Reply:
This sounds like a better solution, will give this a try 👍